blocks.config.json

```diff
@@ -0,0 +1,253 @@
+[
+  {
+    "type": "file",
+    "id": "code-block",
+    "title": "Code block",
+    "description": "A basic code block",
+    "sandbox": false,
+    "entry": "blocks/file-blocks/code/index.tsx",
+    "extensions": ["*"],
```

Comment:
I'd love to move over to the `matches` API in earnest. Thoughts on using this major refactor as an opportunity to deprecate (or at least move away from) `extensions`?

Reply:
agreed, we should take the opportunity to drop `extensions`

Reply:
awsm, dropped!